README.md

```diff
@@ -337,7 +337,7 @@ status:
     ```
 1.  Mount a resource definition value into the binding `Secret`
     ```yaml
-    - path: data.uri
+    - path: data.url
```

Comment:
maybe we should use `uri` here, since that's one of the schema items defined in the spec?